src/BuildScriptGenerator.Common/KeyVaultHelper.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using Azure.Identity;
+using Azure.Security.KeyVault.Secrets;
+
+namespace Microsoft.Oryx.BuildScriptGenerator.Common
+{
+    public static class KeyVaultHelper
+    {
+        public static string GetKeyVaultSecretValue(string keyVaultUri, string secretName)
+        {
+            var client = new SecretClient(new Uri(keyVaultUri), new DefaultAzureCredential());
+            var sasToken = client.GetSecret(secretName).Value.Value;
```

Comment:
try catch the request failed exception that this can throw? [example of this being done](https://msazure.visualstudio.com/One/_git/AAPT-Antares-Containerapps?path=/src/Microsoft.ContainerApps/Microsoft.ContainerApps.Common/Clients/AzureKeyVaultClient/AzureKeyVaultClient.cs&version=GBmain&line=22&lineEnd=31&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents)